src/library_eventloop.js

```diff
@@ -149,12 +149,23 @@ LibraryJSEventLoop = {
     clearInterval(id);
   },
 
+  $registerPostMainLoop: (f) => {
+    // Does nothing unless $MainLoop is included/used.
+    typeof MainLoop != 'undefined' && MainLoop.postMainLoop.push(f);
```

Comment:
```suggestion
    MainLoop?.postMainLoop.push(f);
```

Reply:
Sadly I don't think the ? operator works like that (i.e. it doesn't work on top level things):

```
$ node
Welcome to Node.js v18.20.3.
Type ".help" for more information.
> a?.foo
Uncaught ReferenceError: a is not defined
> 
```

Reply:
Indeed from https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining:

```
Optional chaining cannot be used on a non-declared root object
```

Reply:
Oh sorry. That is unfortunate, I wonder why it has that restriction.